frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.kt

```diff
@@ -275,6 +278,13 @@ class FragNavController constructor(private val fragmentManger: FragmentManager,
             } else {
                 //Attempt to reattach previous fragment
                 fragment = addPreviousFragment(ft, shouldDetachAttachOnSwitch() || shouldRemoveAttachOnSwitch())
+
+                fragmentCache.values.forEach { ref ->
```

Comment:
Should it be an option to set max lifecycle? Without this it will change behaviour of existing apps

Reply:
Right, good catch 👍  Will deal with it